src/client/Microsoft.Identity.Client/ManagedIdentity/ManagedIdentitySource.cs

```diff
@@ -30,38 +31,41 @@ protected ManagedIdentitySource(RequestContext requestContext)
         public virtual async Task<ManagedIdentityResponse> AuthenticateAsync(AppTokenProviderParameters parameters, CancellationToken cancellationToken)
         {
             ManagedIdentityRequest request = CreateRequest(parameters.Scopes.ToArray());
-            
-            var response =
-            HttpMethod.Get.Equals(request.Method) ?
+
+            HttpResponse response =
+            request.Method == HttpMethod.Get ?
             await _requestContext.ServiceBundle.HttpManager.SendGetForceResponseAsync(request.Endpoint, request.Headers, _requestContext.Logger, cancellationToken: cancellationToken).ConfigureAwait(false) :
             await _requestContext.ServiceBundle.HttpManager.SendPostAsync(request.Endpoint, request.Headers, request.BodyParameters, _requestContext.Logger, cancellationToken: cancellationToken).ConfigureAwait(false);
 
             return HandleResponse(parameters, response);
-            
         }
 
         protected virtual ManagedIdentityResponse HandleResponse(
             AppTokenProviderParameters parameters,
             HttpResponse response)
         {
+            string message;
+            Exception exception = null;
+
             try
             {
                 if (response.StatusCode == HttpStatusCode.OK)
                 {
-                    _requestContext.Logger.Info("[Managed Identity] Successful response received.");
+                    _requestContext.Logger.Info($"[{ManagedIdentitySourceName}] Successful response received.");
```

Comment:
nit: This is nice, but it will make it harder to track down in logs :)

Reply:
I will revert it then and use just [Managed Identity]. We do log what type of managed identity source is created.